src/__tests__/fields/contact-field.spec.ts

```diff
@@ -68,6 +69,82 @@ describe("contact-field", () => {
 		});
 	});
 
+	describe("fixed country number", () => {
+		it("should validate specific country", () => {
+			const country = "France" as TCountry;
+
+			const schema = jsonToSchema({
+				section: {
+					uiType: "section",
+					children: {
+						field: {
+							uiType: "contact-field",
+							somethingUnused: "test",
+							validation: [
+								{ contactNumber: { internationalNumber: country }, errorMessage: ERROR_MESSAGE },
+							],
+						},
+					},
+				},
+			});
+
+			expect(() => schema.validateSync({ field: "+33 5-12-34-56-78" })).not.toThrowError();
+		});
+
+		it("should reject invalid numbers", () => {
```

Comment:
- move this test to under `international numbers` test suite
- rename description as `should reject if number format does not match specified country`